docs/developer-guide/nodejs-api.md

```diff
@@ -21,15 +21,15 @@ While ESLint is designed to be run on the command line, it's possible to use ESL
     * [EditInfo type][editinfo]
     * [Formatter type][formatter]
 * [SourceCode](#sourcecode)
-    * [splitLines()](#sourcecode-splitlines)
+    * [splitLines()](#sourcecodesplitlines)
 * [Linter](#linter)
-    * [verify()](#linter-verify)
-    * [verifyAndFix()](#linter-verifyandfix)
-    * [defineRule()](#linter-definerule)
+    * [verify()](#linterverify)
+    * [verifyAndFix()](#linterverifyandfix)
+    * [defineRule()](#linterdefinerule)
     * [defineRules()](#linter-definerules)
-    * [getRules()](#linter-getrules)
-    * [defineParser()](#linter-defineparser)
-    * [version](#linter-version)
+    * [getRules()](#lintergetrules)
+    * [defineParser()](#linterdefineparser)
+    * [version](#linterversionlinterversion)
```

Comment:
`#linter-definerules` should be fixed, too?

Reply:
It's my mistake. I will fix soon.